### ISSUE TITLE
The Clarification of the term 'Broker' in Section 201 is Needed

### ISSUE THREAD

**Issue by OyeOye-12** [responsible-financial-innovation-act22/RFIA-bill#132]
The Prior Infrastructure Bill left the definition of "Broker" very vague and open making almost anyone a "broker".  Better defining this in the Bill is a good start, further clarification may be needed as well to ensure that ordinary users of crypto, and miners, etc... are not considered "brokers" for the purpose of the bill.

------------------

SEC. 202. INFORMATION REPORTING REQUIREMENTS IMPOSED ON BROKERS WITH RESPECT TO DIGITAL ASSETS.

(a) Clarification of Definition of Broker.—Section 6045(c)(1)(D) of the Internal Revenue Code of 1986 is amended to read as follows:

"(D) any person who (for consideration) stands ready in the ordinary course of a trade or business to effect sales of digital assets at the direction of their customers.".

(b) Reporting of Digital Assets.—

(1) Brokers.—

**Comment by steakknife** [comment]
_2-207 enters the chat_

**Comment by kentkos** [comment]
§ 1.6045-1 Returns of information of brokers and barter exchanges.

(a) Definitions. The following definitions apply for purposes of this section and § 1.6045-2:

(1) The term broker means any person (other than a person who is required to report a transaction under section 6043), U.S. or foreign, that, in the ordinary course of a trade or business during the calendar year, stands ready to effect sales to be made by others. A broker includes an obligor that regularly issues and retires its own debt obligations or a corporation that regularly redeems its own stock. However, with respect to a sale (including a redemption or retirement) effected at an office outside the United States, a broker includes only a person described as a U.S. payor or U.S. middleman in § 1.6049-5(c)(5). In addition, a broker does not include an international organization described in § 1.6049-4(c)(1)(ii)(G) that redeems or retires an obligation of which it is the issuer.

**Comment by jonmarrs** [comment]
Perhaps it should be explicitly stated that cryptocurrency miners / node operators / liquidity providers / stakers / hardware suppliers and software developers are not brokers. Essentially, those who provide resources and logistics to support the technical operation, consensus mechanism, data storage/transmission, decentralization, infrastructure, and peripherals of a virtual currency network or blockchain are not considered brokers and are not subject to broker reporting requirements.

**Comment by steakknife** [comment]
Yes they're not brokers unless the ability to broker was decentralizable to
them. The issue is that the *design, development, maintenance* of the
software and/or hardware for economic proxy activity is under-regulated.
For example, does all said hardware meet UL, CE, and/or FCC regs... I
sincerely doubt that all special-purpose hardware isn't going to crash and
catch fire, interfere with communications, or lose value prior to
committing to a blockchain due to bugs.

If you have a credit card, you want to know that there's PCI-DSS behind
that infrastructure and that a bank account is FDIC insured.

If you have a physical electronic wallet, you want to know as a consumer
that it's not going to explode, send its contents to a malicious actor, or
fail entirely. The "invisible hand" of the market is an illusionary hand,
which is why consumer protections and financial regulations need to
properly constrain activities prone to Ponzi schemes, criminal enterprises,
amateur software involving real money, hacking,
interoperability, liquidity, and market stability.

In the sense of "don't roll your own cryptography constructions", "don't
roll your own coin" should be a thing unless it implements and conforms to
basic standards.

Suggestion: a SANS-like reference FOSS crypto library, end-user client, and
server node configurable framework  for proof-of-(W,stake,S,T,S&T), smart
contracts, NFTs complete with deployment, security profiles, and operation
checklists that meet a "Common Criteria EAL4-5"-type standard. A practical
starting point to prevent reinventing-the-wheels involving both
cryptography and money.
